lib/redix/connector.ex

```diff
@@ -190,6 +190,7 @@ defmodule Redix.Connector do
       else
         @default_ssl_opts
       end
+      |> Enum.reject(fn {key, _val} -> Keyword.has_key?(user_socket_opts, key) end)
```

Comment:
You can do

```suggestion
      |> Keyword.drop(Keyword.keys(user_socket_opts))
```

Reply:
That does sound better, didn't know `Keyword.drop/1` :)